tests/cert_v1_unsupported.rs

```diff
@@ -0,0 +1,19 @@
+use std::convert::TryFrom;
+
+// Check with `openssl x509 -text -noout -in cert_v1.pem`
+// to verify this is a correct 1 certificate.
+const CERT_V1_PEM: &str = include_str!("cert_v1.pem");
+
+#[test]
+fn error() {
+    let mut lines: Vec<&str> = CERT_V1_PEM.lines().collect();
+    assert_eq!(lines.first().unwrap(), &"-----BEGIN CERTIFICATE-----");
+    assert_eq!(lines.last().unwrap(), &"-----END CERTIFICATE-----");
+    lines.remove(0);
+    lines.pop().unwrap();
+    let der = base64::decode(&lines.join("")).unwrap();
```

Comment:
Thanks. Instead of doing this base64/PEM stuff, just add the binary (DER) certificate instead, like other tests do.

Reply:
I'll convert to der.